examples/explorer/src/helpers/datetimeHelper.js

```diff
@@ -0,0 +1,35 @@
+import { networkSettings } from '../config.json';
+
+const { genesisTimestamp, slotsPerEpoch, slotDuration } = networkSettings;
+
+/**
+ * This functions gets a BlockDate and returns the amount of
+ * seconds past since genesis block
+ * @param blockDate Formed by an epoch object and a slot number
+ */
+const secondsSinceGenesis = ({ epoch, slot }) => {
+  // Epoch id is the Epoch number starting at zero.
+  const epochCount = Number.parseInt(epoch.id, 10);
+  const slotCount = Number.parseInt(slot, 10);
+  const slotsSinceGenesis = slotsPerEpoch * epochCount + slotCount;
```

Comment:
@NicolasDP can we have any problem by doing this? Something that can make this way off?